ui/src/contexts/FeatureFlagsContext.ts

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+interface FeatureFlags {
```

Comment:
does this let users flip it on / off in the top level FeastUI component?

Reply:
Yup

Reply:
See this comment: https://github.com/feast-dev/feast/pull/2370#discussion_r825364792